bedrock/security/templates/security/bug-bounty/faq-webapp.html

```diff
@@ -113,34 +112,150 @@ <h1 itemprop="name" class="title-shadow-box">Web Application Security Bug Bounty
         <dt class="question" id="eligible-bugs">Which domains and web applications
           will be considered to be part of the bug bounty?</dt>
         <dd class="answer">
-          <p>Below is the list of domains under scope for Mozilla’s web application
-            security
-            bug bounty. While this list doesn’t include all the Mozilla web properties,
-            we do plan on adding to this list as we introduce our web security process to
-            other sites not included in this list.</p>
-
-          <ul>
-            <li>bugzilla.mozilla.org</li>
-            <li>*.services.mozilla.com</li>
-            <li>aus*.mozilla.org</li>
-            <li>www.mozilla.com/org</li>
-            <li>www.firefox.com</li>
-            <li>www.getfirefox.com</li>
-            <li>addons.mozilla.org</li>
-            <li>services.addons.mozilla.org</li>
-            <li>versioncheck.addons.mozilla.org</li>
-            <li>download.mozilla.org</li>
-          </ul>
-
-          <p>Additionally, Mozilla services that handle reasonably sensitive user data
-            such as Sync and Hello are in scope for the bug bounty program.</p>
-
-          <p>The Mozilla Developer Network (developer.mozilla.org) has been removed from this
-            list due to site vandalism by researchers looking for security issues. It will be
+          <p>Below is the list of domains eligible for Mozilla’s website and services
+            security bug bounty. Please check it regularly, as it changes over time.</p>
+
+          <h4>www.mozilla.com/org</h4>
+          <ul>
+          	<li>www.mozilla.com</li>
+          	<li>www.mozilla.org</li>
+          	<li>www.firefox.com<li>
```

Comment:
:rage3:

/me falls down

Fixed directly on master in d7580090e46a0fba598a1f096c9c1c76c466708a.